### PR TITLE
Change the readme contents

### PR DESCRIPTION
**Pull request by rasanjaya85**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # terraform-apim
-Terraform scripts for provision the APIM deployments on Azure cloud
+Terraform scripts for provision the APIM deployments on Azure cloud.
```
Comment (chirangaalwis) [inline README.md:2]: @rasanjaya85 this has to be `provisioning`.

File: azure-simple/Packer/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
```
Comment (chirangaalwis) [inline azure-simple/Packer/Vagrantfile:2]: @rasanjaya85 please update the license header year. There are other applicable files as well.

File: azure-simple/Packer/ansible-apim/dev/inventory [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[apim]
+apim_1 ansible_host=localhost ansible_user=centos ansible_connection=local
```
Comment (chirangaalwis) [inline azure-simple/Packer/ansible-apim/dev/inventory:2]: @rasanjaya85 please add a new EOF line.

File: azure-simple/Packer/ansible-apim/docs/Pattern_1.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+```
+API Manager pattern 1 contains 2 groups and the configurations specific for each group should be in the respective yaml file under [group_vars](../dev/group_vars) folder. Configurations specific to each host should be added to the corresponding yaml file under [host_vars](../dev/host_vars) folder.
+
+Most commonly changed values are parameterized in the above files. If further changes are required, the values should be parameterized and added to the files accordingly.
+
+#### i. Customize `apim` role
+
+Navigate to [carbon-home](../roles/apim/templates/carbon-home) of the `apim` role. All the files required to deploy the API Manager active-active combination are here. Follow the instructions in the following document to modify the files.
+- [Configuring an active-active deployment](https://docs.wso2.com/display/AM260/Configuring+an+Active-Active+Deployment)
```
Comment (chirangaalwis) [inline azure-simple/Packer/ansible-apim/docs/Pattern_1.md:86]: @rasanjaya85 this refers to older documentation. Please update to [this](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/single-node/configuring-an-active-active-deployment/).

File: azure-simple/Packer/ansible-apim/docs/Pattern_1.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+#### ii. Customize `apim-analytics-worker` role
+
+Navigate to [carbon-home](../roles/apim-analytics-worker/templates/carbon-home) of the `apim-analytics-worker` role. All the files required to deploy the API Manager analytics are here. Follow the instructions in the following files to modify the files.
+- [Configure Analytics](https://docs.wso2.com/display/AM260/Configuring+APIM+Analytics#standardsetup)
```
Comment (chirangaalwis) [inline azure-simple/Packer/ansible-apim/docs/Pattern_1.md:91]: @rasanjaya85 this refers to older documentation. Please update to [this](https://apim.docs.wso2.com/en/latest/learn/analytics/configuring-apim-analytics/#standard-setup).

File: azure-simple/Packer/ansible-apim/docs/Pattern_1.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+- [Minimum HA deployment](https://docs.wso2.com/display/SP430/Minimum%20High%20Availability%20Deployment)
```
Comment (chirangaalwis) [inline azure-simple/Packer/ansible-apim/docs/Pattern_1.md:92]: @rasanjaya85 should refer to [this](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/distributed-deployment/configure-apim-analytics/active-passive/).

File: azure-simple/Packer/ansible-apim/roles/apim/tasks/main.yml [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+      template:
+        src: wso2apim.service.j2
+        dest: /etc/systemd/system/wso2apim.service
+
+    - name: Stop wso2apim as a service
+      systemd:
+        name: wso2apim
+        state: stopped
+
+    - name: Check if deployment resource directory exists
```
Comment (chirangaalwis) [inline azure-simple/Packer/ansible-apim/roles/apim/tasks/main.yml:53]: @rasanjaya85 just out of curiosity. Are we sharing the entire `<AM_HOME>/repository/deployment/server` directory in this case? As per the team's recommendation, we need to share only the specific directories such as, `executionplans`, `synapse-configs` and `userstores` (optional according to the product team and may not be required in the default installation artifacts).

File: azure-simple/Packer/centos-base.json [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+      "execute_command": "echo '{{user `ssh_password`}}' | {{ .Vars }} sudo -S -E sh '{{ .Path }}'",
+      "inline": [
+          "bash /tmp/scripts/init.sh",
+          "/usr/sbin/waagent -force -deprovision+user && export HISTSIZE=0 && sync"],
+      "inline_shebang": "/bin/sh -x",
+      "type": "shell",
+      "skip_clean": true
+    }
+  ]
+}
```
Comment (chirangaalwis) [inline azure-simple/Packer/centos-base.json:63]: @rasanjaya85 please add a new EOF line.

File: azure-simple/Packer/variables.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+  "azure-client-id": "*********************",
+  "azure-client-secret": "*********************",
+  "azure-tenant-id": "*********************",
+
+  "azure-resource-group": "WSO2-installers",
+  "azure-region": "East US",
+
+  "vm-size": "Standard_A1_v2",
+  "storage-account" : "wso2installer"
+}
```
Comment (chirangaalwis) [inline azure-simple/Packer/variables.json:12]: @rasanjaya85 please add a new EOF line.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

**Comment by chirangaalwis** [comment]
@rasanjaya85 it would be great if we can create issues for this and link to this PR accordingly.

File: azure-simple/Packer/ansible-apim/roles/apim/tasks/main.yml (hunk 7 above)
Comment (rasanjaya85) [reply azure-simple/Packer/ansible-apim/roles/apim/tasks/main.yml:53]: No, This release validates the  "{{ carbon_home }}/repository/deployment/server" is exist then synchronize only the "{{ carbon_home }}/repository/deployment/server/synapse-configs" files to mounted data disk. This behavior has taken from the Cloudformation scripts(Pattern1) and converted into the Ansible way to do that.

**Comment by rasanjaya85** [comment]
Closed

**Pull request closed without merging**